packages/store/src/storage.ts

```diff
@@ -2068,20 +2068,21 @@ class Storage {
     // Checks to perform if config "publish:check_owners" is true
     debug('check if user %o is an owner and allowed to change package', username);
     // if name of owner is not included in list of maintainers, then throw an error
+    // packages owned by anonymous user are allowed to be changed by any user
     if (
       this.config?.publish?.check_owners === true &&
       manifest.maintainers &&
       manifest.maintainers.length > 0 &&
       !manifest.maintainers.some((maintainer) => {
         if (typeof maintainer === 'string') {
-          return maintainer === username;
+          return maintainer === username || maintainer === DEFAULT_USER;
```

Comment:
It might `DEFAULT_USER = 'Anonymous';` not to be a good feed here, it is meant to be used as label for the UI.